godot_ap/ui/console/base_console.gd

```diff
@@ -14,6 +14,8 @@
 			return spacing
 		return parts_cont.get_theme_constant(&"separation")
 
+static var console_label_fonts: FontStorage
```

Comment:
Only one thing here... if this is static anyway, why is it being passed around everywhere? It'd be better to NOT pass it to all the functions and stuff, and just use `BaseConsole.console_label_fonts` in whatever files you need it?

Reply:
makes sense, will fix